source/postprocess/global_statistics.cc

```diff
@@ -176,7 +176,7 @@ namespace aspect
             statistics.add_value("Number of nonlinear iterations",
                                  nonlinear_iterations);
 
-          // Only output statistics columns if the solver actually signaled at least one
+          // Only output statistics columns if the solver actually signalled at least one
```

Comment:
I really had to look this one up because I would have spelled it with two L as well...